source/enemies/elegantstella.js

```diff
@@ -57,13 +57,13 @@ module.exports = new EnemyTemplate("Elegant Stella",
 	description: `Deal minor ${getEmoji("Light")} damage and slowing @e{Misfortune} to all foes`,
 	priority: 0,
 	effect: (targets, user, adventure) => {
-		const pendingMisfortune = 1;
+		let pendingMisfortune = 1;
 		if (user.crit) {
 			pendingMisfortune += 7;
 		}
```

Comment:
Good find on this bug, here's the way I usually write this pattern:
```suggestion
		const pendingMisfortune = { name: "Misfortune", stacks: 1 };
		if (user.crit) {
			pendingMisfortune.stacks += 7;
		}
```